storm-core/src/clj/backtype/storm/ui/core.clj

```diff
@@ -951,17 +958,17 @@
   (GET "/api/v1/topology/summary" [:as {:keys [cookies servlet-request]} & m]
        (assert-authorized-user servlet-request "getClusterInfo")
        (json-response (all-topologies-summary) (:callback m)))
-  (GET  "/api/v1/topology/:id" [:as {:keys [cookies servlet-request]} id & m]
+  (GET  "/api/v1/topology/:id" [:as {:keys [cookies servlet-request schema]} id & m]
```

Comment:
@knusbaum 
I don't know about compojure / ring, and I can't find related docs, but core.clj from master branch uses different key name.
Actually master branch uses both schema and scheme.

- route which uses schema as key: https://github.com/apache/storm/blob/master/storm-core/src/clj/backtype/storm/ui/core.clj#L997
- route which uses scheme as key: https://github.com/apache/storm/blob/master/storm-core/src/clj/backtype/storm/ui/core.clj#L1004

Which one is correct? Or both things are correct?